.github/workflows/base.yml

```diff
@@ -344,18 +363,47 @@ jobs:
           cd quarkus
           bash ../workflow-mandrel/.github/update_quarkus_version.sh 2.2.999
         fi
-    # Use Java 17 to build Quarkus as it doesn't build with Java 20
+    # Use Java 11 to build Quarkus as that's the lowest supported JDK version currently
     - uses: actions/setup-java@v3
       with:
         distribution: 'temurin'
-        java-version: '17'
+        java-version: '11'
+    - name: Download GraalVM Maven Repo
+      if: "!startsWith(matrix.os-name, 'windows')"
```

Comment:
There is no matrix in this job, this is always run on linux

```suggestion
```

Reply:
Fixed, thanks!